data.py-dist

```diff
@@ -50,14 +50,28 @@ VM_IMAGES = {
 }
 
 # FIXME: use URLs and an optional cache?
+#
+# If net-only is set to True only URL source will be possible. By default the
+# parameter is set to False so local source and url, if one is defined
+# in NETINSTALL_URL, are used.
 ISO_IMAGES = {
-#    'xcpng-8.0': "/home/user/iso/xcp-ng-8.0.0.iso",
-#    'xcpng-8.1': "/home/user/iso/xcp-ng-8.1.0-2.iso",
-#    'xcpng-8.2.0': "/home/user/iso/xcp-ng-8.2.0.iso",
-#    'xcpng-8.2.1': "/home/user/iso/xcp-ng-8.2.1.iso",
-#    'xcpng-8.2.1-2023': "/home/user/iso/xcp-ng-8.2.1-20231130.iso",
-#    'xcpng-8.3-beta2': "/home/user/iso/xcp-ng-8.3.0-beta2.iso",
-#    'xcpng-8.3-beta2-net': "/home/user/iso/xcp-ng-8.3.0-beta2-netinstall.iso",
+#    'xcpng-8.0': {'path': "/home/user/iso/xcp-ng-8.0.0.iso"},
+#    'xcpng-8.1': {'path': "/home/user/iso/xcp-ng-8.1.0-2.iso"},
+#    'xcpng-8.2.0': {'path': "/home/user/iso/xcp-ng-8.2.0.iso"},
+#    'xcpng-8.2.1': {'path': "/home/user/iso/xcp-ng-8.2.1.iso"},
+#    'xcpng-8.2.1-2023': {'path': "/home/user/iso/xcp-ng-8.2.1-20231130.iso"},
+#    'xcpng-8.3-beta2': {'path': "/home/user/iso/xcp-ng-8.3.0-beta2.iso"},
+#    'xcpng-8.3-beta2-net': {
+#       'path': "/home/user/iso/xcp-ng-8.3.0-beta2-netinstall.iso",
+#       'net-only': True,
```

Comment:
maybe instead of putting the URL in NETINSTALL_URL we can put it here. Something like:
```
#    'xcpng-8.2.1-2023': {'path': "/home/user/iso/xcp-ng-8.2.1-20231130.iso"},
#    'xcpng-8.3-beta2': {
#       'path': "/home/user/iso/xcp-ng-8.3.0-beta2.iso",
#       'url': "http://server/installers/xcp-ng/8.3-beta2",
#    },
#    'xcpng-8.3-beta2-net': {
#       'path': "/home/user/iso/xcp-ng-8.3.0-beta2-netinstall.iso",
#       'net-only': True,
#       'url': "http://server/installers/xcp-ng/8.3-beta2",
#    },
```
In this example  'xcpng-8.2.1-2023' will be able to run local install, 'xcpng-8.3-beta2' will be able to run both (local and netinstall) and 'xcpng-8.3-beta2-net' will be only netinstall...

Reply:
I finally use `path`, `net-url`, `net-only` and `pxe-url` and removed the NETINSTALL_URL. I think it is cleaner this way.